probe-rs/src/debug/debug_info.rs

```diff
@@ -318,6 +320,8 @@ impl DebugInfo {
         let unit_header = self.dwarf.debug_info.header_from_offset(header_offset)?;
         let unit_info = UnitInfo::new(gimli::Unit::new(&self.dwarf, unit_header)?);
 
+        let mut type_cache = TypeCache::new();
```

Comment:
If this type is indeed a cache, wouldn't we be able to speed things up by not recreating an empty one for calls to this fn?

Reply:
Yes, that's the idea. Just didn't get that far yet.